test/datatypes.test.js

```diff
@@ -57,6 +57,17 @@ describe('MySQL specific datatypes', function() {
       Account.destroyAll(done);
     });
 
+    it('discover type of amount', function(done) {
+      db.discoverModelProperties({modelName: 'Account'}, function(err, result) {
```

Comment:
This is not valid.
```js
db.discoverModelProperties(<model_name>, options, cb);
```
More about it on the [apidocs](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-discovermodelproperties).